jck/jtrunner/JavaTestRunner.java

```diff
@@ -1334,11 +1340,35 @@ private static boolean isIbmJvm() {
 		return javaVMName.contains("ibm") || javaVMName.contains("openj9"); 
 	}
 	
+	/**
+	 * Return the jckVersion minus any letters.
+	 */
 	private static int getJckVersionInt(String version) {
-		if (version.equals("8c") || version.equals("8b")) {
-			return 8; 
-		} else {
-			return Integer.parseInt(version); 
+		String subVersion = version;
+		if (version.matches("^jck[0-9]+[a-z]*$")) {
```

Comment:
Is this part needed given that the next part will strip off `^jck` anyway?

Reply:
True. I've adapted this bit and the bit in the next method.